resources/targets/simple.yaml

```diff
@@ -1,3 +1,7 @@
+-
+    name: Kepler 488 b
```

Comment:
One issue we need to deal with is consistency in naming and fields. These automatically get uploaded to the storage buckets with these names.

I have been putting the name of the host star, not the planet itself. So just `Kepler 488`. This could feasibly change in the future when you have a timed observation and are going after a different exoplanet but until then I think we should just list the host star.

Reply:
Will do.